cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,12 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
+  const email = 'mlemeshev@i.ua';
+  const password = 'User1234';
```

Comment:
The test data (email and password) are hardcoded into your test. This could be a problem if you need to change the test data, as you would have to change it in every test that uses it. Consider moving this to a separate test data file or at least to a higher scope to make it reusable and easy to manage.